metrics/cloudwatch/cloudwatch.go

```diff
@@ -271,10 +271,6 @@ func sum(a []float64) float64 {
 	return v
 }
 
-func last(a []float64) float64 {
-	return a[len(a)-1]
-}
-
```

Comment:
This should probably be a separate commit/PR.

Reply:
Done